{"name": "test_update_post_with_invalid_id", "status": "skipped", "statusDetails": {"message": "XFAIL Fake API accepts string ID\n\nAssertionError: Expected 400 or 404 for invalid ID\nassert 500 in [400, 404]\n +  where 500 = <Response [500]>.status_code", "trace": "api = <api.posts_api.PostsAPI object at 0x0000024FA47DEB90>\n\n    @pytest.mark.negative_test\n    @pytest.mark.xfail(reason=\"Fake API accepts string ID\")\n    def test_update_post_with_invalid_id(api):\n        payload = negative_data[\"invalid_id_post\"]\n        res = api.update_post(\"abc\", payload)\n>       assert res.status_code in [400, 404], \"Expected 400 or 404 for invalid ID\"\nE       AssertionError: Expected 400 or 404 for invalid ID\nE       assert 500 in [400, 404]\nE        +  where 500 = <Response [500]>.status_code\n\ntests\\test_posts.py:82: AssertionError"}, "attachments": [{"name": "log", "source": "965e45f1-4038-4065-860e-d3972a9d5dbd-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "aefb1ee5-23b5-49b1-8f5e-7f7ce7bbf81e-attachment.txt", "type": "text/plain"}], "start": 1751377539401, "stop": 1751377540050, "uuid": "849e51cc-8a05-439b-a71a-29033e341c72", "historyId": "1c5e41e8af2790ca3aaa52307104bd68", "testCaseId": "1c5e41e8af2790ca3aaa52307104bd68", "fullName": "tests.test_posts#test_update_post_with_invalid_id", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(reason='Fake API accepts string ID')"}, {"name": "tag", "value": "negative_test"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_posts"}, {"name": "host", "value": "DESKTOP-J0TB641"}, {"name": "thread", "value": "6440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_posts"}]}